### PR TITLE
Bump vagrant boxes to fedora37

### PR DESCRIPTION
**Pull request by dereknola**
Signed-off-by: Derek Nola <derek.nola@suse.com>

#### Proposed Changes ####
- Bump to fedora37 for vagrant boxes
- Disable rootless sonobuoy testing, will be enabled in a future PR. 

#### Types of Changes ####
Testing Bump

#### User-Facing Change ####

```release-note

```

#### Further Comments ####
- Fedora has proven too volatile for cgroups and rootless testing. I would like to move to Ubuntu for this testing and will be opeing a PR to add that within the week.

### CODE REVIEW

File: tests/TESTING.md [hunk 1]
```diff
@@ -66,13 +66,13 @@ The sub-directories therein contain fixtures for running simple clusters to asse
...
   - [Rocky Linux 8](../tests/install/rocky-8) (stand-in for RHEL 8)
-  - [Fedora](../tests/install/fedora)
+  - [Fedora 37](../tests/install/fedora)
   - [Leap 15.4](../tests/install/opensuse-leap) (stand-in for SLES)
   - [MicroOS](../tests/install/opensuse-microos) (stand-in for SLE-Micro)
   - [Ubuntu 20.04](../tests/install/ubuntu-focal) (Focal Fossa)
 - [Control Groups](../tests/cgroup) :arrow_right: on any code change
   - [mode=unified](../tests/cgroup/unified) (cgroups v2)
-    - [Fedora 35](../tests/cgroup/unified/fedora-35) (rootfull + rootless)
+    - [Fedora 37](../tests/cgroup/unified/ fedora) (rootfull + rootless)
```
Comment (brandond) [inline tests/TESTING.md:75]: nit?
```suggestion
    - [Fedora 37](../tests/cgroup/unified/fedora) (rootfull + rootless)
```

File: .github/workflows/cgroup.yaml [hunk 2]
```diff
@@ -46,7 +46,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        vm: [fedora-35]
+        vm: [ fedora ]
```
Comment (brandond) [inline .github/workflows/cgroup.yaml:49]: nit?
```suggestion
        vm: [fedora]
```

**Review by brandond — APPROVED** [review]

**Review by briandowns — APPROVED** [review]

**Review by cwayne18 — APPROVED** [review]
Lgtm thanks!

**Comment by brandond** [comment]
Also needs backporting to release-1.23

File: .github/workflows/cgroup.yaml [hunk 3]
```diff
@@ -94,6 +94,6 @@ jobs:
       - name: "[Rootless] K3s Ready"
         env: {TEST_KUBECONFIG: /home/vagrant/.kube/k3s.yaml}
         run: vagrant provision --provision-with=k3s-ready
-      - name: "[Rootless] Sonobuoy (--mode=quick)"
-        env: {TEST_KUBECONFIG: /home/vagrant/.kube/k3s.yaml, TEST_RESULTS_PATH: rootless}
-        run: vagrant provision --provision-with=k3s-sonobuoy
+      # - name: "[Rootless] Sonobuoy (--mode=quick)"
+      #   env: {TEST_KUBECONFIG: /home/vagrant/.kube/k3s.yaml, TEST_RESULTS_PATH: rootless}
+      #   run: vagrant provision --provision-with=k3s-sonobuoy
```
Comment (AkihiroSuda) [inline .github/workflows/cgroup.yaml:99]: Could you open an issue with failure logs?

**Pull request merged**